From 2a20064e44d14b2c84bf52fde9dfd3b43832a74b Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 16 May 2014 13:32:47 +0200
Subject: [PATCH] ifcfg/write-ifcfg.sh: include net-lib.sh

otherwise is_persistent_ethernet_name() is missing

(cherry picked from commit 8fa1afb65cef30d56ce2e141b0b594cf780120a1)
---
 modules.d/45ifcfg/write-ifcfg.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/45ifcfg/write-ifcfg.sh b/modules.d/45ifcfg/write-ifcfg.sh
index 457a9ac4..7f997a2a 100755
--- a/modules.d/45ifcfg/write-ifcfg.sh
+++ b/modules.d/45ifcfg/write-ifcfg.sh
@@ -3,6 +3,8 @@
 # ex: ts=8 sw=4 sts=4 et filetype=sh
 
 # NFS root might have reached here before /tmp/net.ifaces was written
+type is_persistent_ethernet_name >/dev/null 2>&1 || . /lib/net-lib.sh
+
 udevadm settle --timeout=30
 
 if [ -e /tmp/bridge.info ]; then
